server/services/team/service.ts

```diff
@@ -32,6 +33,7 @@ export class TeamService extends ServiceClass implements ITeamService {
 	private MessagesModel: MessagesRaw;
 
 	// TODO not sure we should have the collection here or call the Room service for getting Room data
+	// What's the difference between this one and the one on L25?
```

Comment:
It's duplicated, you can remove this one.

Reply:
Nice, thanks